tests/numpy/pendulum.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
```

Comment:
We cannot accept manual tests anymore. Every new unit test should be run automatically by Pytest.